### PR TITLE
Feature/itm 340

### PR DESCRIPTION
**Pull request by mrb-itminds**
depends on #77 
The flow for creating a new AppToken -> Browsing Services and reqesting acces -> Seeing status of request -> Generating the JWT from the auth-service..
New query to fetch appToken by Id to avoid fetching all tokens when only editing 1.

### CODE REVIEW

File: frontend/src/components/ApplicationScreen/Tokens/TokenStatus/TokenStatusList.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  const [services, setTokenServices] = useState<number[]>([]);
+
+  /**
+   * Used to divide the different actions into the respective services for
+   */
+  useEffect(() => {
+    const services = currToken.appTokenActions.map(action => {
+      return action.action.serviceId;
+    });
+    const set = new Set(services);
```
Comment (objmj-itminds) [inline frontend/src/components/ApplicationScreen/Tokens/TokenStatus/TokenStatusList.tsx:19]: While this is a nifty solution. It isn't in big O time.

```
currToken.appTokenActions.map(action => action.action.serviceId).filter((item, I, arr) => i === arr.indexOf(item))
```

File: frontend/src/components/Forms/Application/AppTokenForm.tsx [hunk 2]
```diff
@@ -61,11 +59,11 @@ const AppTokenForm: FC<Props> = ({ submitCallback, AppMetaData }) => {
                 />
               </FormControl>
               {isLoading ? (
-                <Button variant="outline" width="full" mt={6}>
+                <Button bgColor="blue.200" variant="outline" width="full" mt={6}>
```
Comment (objmj-itminds) [inline frontend/src/components/Forms/Application/AppTokenForm.tsx:62]: colorscheme or useColor. Don't hardcode colors on buttons

File: frontend/src/components/Forms/Application/AuthTokenForm.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  const toast = useToast();
+
+  const CreateJWT = useCallback(async () => {
+    const applicationClient = await genApplicationClient();
+    try {
+      const appResult = await applicationClient.createAuthAppToken(
+        aid,
+        new CreateAuthAppTokenCommand({
+          tokenInput: new TokenInput({
+            tokenIdentifier: "bla bla",
```
Comment (objmj-itminds) [inline frontend/src/components/Forms/Application/AuthTokenForm.tsx:48]: ?

File: frontend/src/components/Forms/Application/AuthTokenForm.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      setIsLoading(false);
+      onOpen();
+    },
+    [appSecret]
+  );
+  return (
+    <Center>
+      <Wrap width="full" justify="center">
+        <Flex width="full" align="center" justifyContent="center">
+          <Box width="full" p={6}>
```
Comment (objmj-itminds) [inline frontend/src/components/Forms/Application/AuthTokenForm.tsx:79]: That's a lot of flexing. Why?

**Review by objmj-itminds — CHANGES_REQUESTED** [review]

File: frontend/src/components/Forms/Application/AuthTokenForm.tsx (hunk 3 above)
Comment (mrb-itminds) [reply frontend/src/components/Forms/Application/AuthTokenForm.tsx:48]: The identifiers are first added in branch 361, so AppToken currently has no identifier to write here. Should properly have added a //TODO or something so it was clearer.

File: frontend/src/components/Forms/Application/AuthTokenForm.tsx (hunk 4 above)
Comment (mrb-itminds) [reply frontend/src/components/Forms/Application/AuthTokenForm.tsx:79]: Tbh. I have no idea.. I noticed in a later branch and wondered the same thing.
I've removed the clutter 👍

File: frontend/src/components/Forms/Application/AppTokenForm.tsx (hunk 2 above)
Comment (mrb-itminds) [reply frontend/src/components/Forms/Application/AppTokenForm.tsx:62]: Understood. branch 361 comes with lots of fixes to bad color practice throughout the application as well as proper use of flex/containers for better responsiveness.

File: frontend/src/components/ApplicationScreen/Tokens/TokenStatus/TokenStatusList.tsx (hunk 1 above)
Comment (mrb-itminds) [reply frontend/src/components/ApplicationScreen/Tokens/TokenStatus/TokenStatusList.tsx:19]: Yea I can see your solution is more optimal.

**Pull request merged**